dashboard/config/initializers/cors.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
```

Comment:
It's great that this uses Rack middleware. that will make it easy to add to pegasus to implement CORS for /v2/ routes.

Reply:
It might be useful to insert this rack middleware explicitly in our Rails `application.rb` so that we’re very explicit about what order we’re loading each middleware component.
https://www.seattle.gov/documents/Departments/CityFinance/FilingADamageClaim/Seattle-claim-for-damages-form.pdf

Reply:
I think I did what you had in mind in 34b13f1e42f. 

The headers look the same as before:

![Screenshot 2024-01-31 at 2 53 43 PM](https://github.com/code-dot-org/code-dot-org/assets/46464143/363c3c8e-a291-4609-866b-b10bfeaacd33)

Reply:
Given how many items we're inserting into the middleware, I think our rack functionality would be easier to maintain if we loaded the CORS middleware in `application.rb` so everyone can see what order we're loading everything and why.